test/jasmine/assets/domain_ref/testnumber.js

```diff
@@ -0,0 +1,3 @@
+// module.exports can be set to a specific test number to run just a single
+// test, otherwise if set to undefined, runs all the tests.
+module.exports = undefined;
```

Comment:
Sorry. I don't understand if we really need this to be a separate file?

Reply:
It's there because jasmine can't read environment variables which would be handy to pin point a specific test that is not passing. So instead I made a file that you change. But I can remove this now because the tests pass.

Reply:
@nicholas-esterer have you seen `fit` and `fdescribe`? https://davidtang.io/2016/01/03/controlling-which-tests-run-in-jasmine.html

Reply:
Thanks, those are handy but in this case I had a for loop that generates many `it(...)` so if I changed it to `fit(...)` it would disable all the tests. Plus I personally don't like changing the source file, environment variables are more ephemeral and the risk of them getting committed is lower. But thanks for the tip!